src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Thank you) Yes , I can create a single instance of Random for the entire class .it reduces the overhead of object creation and initialization. Reusing the same Random instance ensures consistent behavior across multiple method invocations. The Random class is not thread-safe, meaning it may produce unpredictable behavior if accessed concurrently by multiple threads.

Reply:
I guess thread safety would be a problem for another day then :)
But we talk about it seriously, usually applications are either fully multhithreading oriented, safe and designed with a thought about it from the beginning of the development, or just a simpler apps that can partially use some parallelism but that do not support that fully and should be refactored if we want to accommodate for that.